examples/hse/lcao_Si2/INPUT

```diff
@@ -10,12 +10,12 @@ ks_solver  				genelpa
 ecutwfc    				100
 scf_thr        				1e-8
 scf_nmax      				100
-gamma_only 				0
+gamma_only 				1
 smearing_method   			gauss
 mixing_type				pulay
 mixing_beta             0.4
 
-dft_functional			hse
+dft_functional			scan
```

Comment:
Maybe you can add a new scan example, and keep the hse example unchanged.

Reply:
This is mistakenly modified and has been reverted.